docs/advanced-topics/stale-index.md

```diff
@@ -0,0 +1,47 @@
+# Permanent stale index inconsistency
```

Comment:
@porunov It seems you forgot to add this doc to the nav config. As a result, this page won't be visible to users.